.eslintignore

```diff
@@ -0,0 +1,3 @@
+test/migrations/generated/
```

Comment:
Any reason the migrations shouldn't pass eslint?

Reply:
The eslint violations are minor (missing semi at the end of a chain and newline first line of func)

Reply:
Ah, got it!  So I assume you plan to fix those lint issues?  In that case can we remove this .eslintignore so that we can detect if the generated files do not pass lint.

Reply:
Yes I'll fix this today